plugins/maciVoting/services/coordinator/utils/errors.ts

```diff
@@ -0,0 +1,33 @@
+/** This entire file is copied from https://github.com/privacy-scaling-explorations/maci/blob/dev/apps/coordinator/ts/common/errors.ts */
```

Comment:
I think we can delete this for the moment :) 

Just hard code the `ErrorCodes` we throw in the frontend

Reply:
Done